dnf-behave-tests/dnf/plugins-core/download-debuginfo.feature

```diff
@@ -1,6 +1,5 @@
-@xfail
-# The option for download command is missing:
-# https://github.com/rpm-software-management/dnf5/issues/948
+# Tests for the dnf download command with --debuginfo and --debugsource options
+# Implementation: https://github.com/rpm-software-management/dnf5/issues/948
 Feature: dnf download --debuginfo command
```

Comment:
The file and the feature mention only debuginfo, but the comment says also debugsource and the commit message mentions only debugsource. With the other PR, https://github.com/rpm-software-management/ci-dnf-stack/pull/1799 , I understand that you plan to add a separate file for debugsource tests. In that case, since I am reviewing debuginfo first and conflicts will need to be resolved anyway, maybe you could create the file already in this PR? In any case, the commit message should be fixed.